submissions/BogdanGapon/HTML & CSS popup/index.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="container">
+        <nav class="header-nav">
+          <ul class="header-nav__list">
+            <li class="header-nav__item">
+              <a href="" class="header-nav__link">Mail</a>
+            </li>
+            <li class="header-nav__item">
+              <a href="" class="header-nav__link" target="_blank">Images</a>
+            </li>
+            <li class="header-nav__item popup-item">
+              <label class="custom-checkbox" for="check"></label>
+              <input
+                class="checkbox visually-hidden"
+                type="checkbox"
+                id="check"
+              />
+              <span class="span"></span>
+              <div class="popup-wrapper">
+                <div class="popup-flex-wrapper">
+                  <a href="" class="popup-list__link"
+                    ><img
+                      src="./img/apps.png"
+                      alt="Colorful circle"
+                      width="45"
+                      height="45"
+                    />Colorful circle</a
```

Comment:
It is really great that you have fount that attributes width and height could be passed to img tag. But they are using, when css styles are don't. So if you use css styles this attributes using is incorrect for html and css separation logic.
Please, move this styles into css.
![image](https://user-images.githubusercontent.com/42061361/181111952-dc408cfc-d02c-412e-9191-c62093129f17.png)

Reply:
"But they are using, when css styles are don't." I don`t use css styles for img (width and height) only responsible styles like  max-width: 100%; and that is all.

Reply:
But you use css files. This what i mean.  You can't swith between this two cases as you want. If you are using css files, you hade better not use this attributes.